sdk/core/azure-core/src/main/java/com/azure/core/util/CoreUtils.java

```diff
@@ -29,6 +30,7 @@
  * This class contains utility methods useful for building client libraries.
  */
 public final class CoreUtils {
+    private static final ClientLogger LOGGER = new ClientLogger(CoreUtils.class);
```

Comment:
What is using this logger? I see it is a newly added property but no new code uses it.